analytics/views/realm_activity.py

```diff
@@ -71,6 +81,12 @@ def is_recent(val: Optional[datetime]) -> bool:
         email_link = user_activity_link(email)
         sent_count = get_count(user_summary, "send")
         cells = [user_summary["name"], email_link, sent_count]
+        role = user_summary["role"]
+        cells = [user_summary["name"], email_link, role, sent_count]
+
+        if "bot_owner" in user_summary:
+            cells[2] = get_bot_owner_email(user_summary)
```

Comment:
nit -- I'd inline this logic:
```suggestion
        cells = [
            user_summary["name"],
            email_link,
            role if "bot_owner" not in user_summary else get_bot_owner_email(user_summary),
            sent_count,
        ]